packages/client/components/AddTeamDialog.tsx

```diff
@@ -0,0 +1,201 @@
+import React, {useState} from 'react'
+import {PreloadedQuery, usePreloadedQuery, useFragment} from 'react-relay'
+import PrimaryButton from './PrimaryButton'
+import SecondaryButton from './SecondaryButton'
+import {Dialog} from '../ui/Dialog/Dialog'
+import {DialogContent} from '../ui/Dialog/DialogContent'
+import {DialogTitle} from '../ui/Dialog/DialogTitle'
+import {DialogActions} from '../ui/Dialog/DialogActions'
+import {Select} from '../ui/Select/Select'
+import {SelectItem} from '../ui/Select/SelectItem'
+import {SelectTrigger} from '../ui/Select/SelectTrigger'
+import {SelectGroup} from '../ui/Select/SelectGroup'
+import {SelectValue} from '../ui/Select/SelectValue'
+import {SelectContent} from '../ui/Select/SelectContent'
+import graphql from 'babel-plugin-relay/macro'
+import {AddTeamDialogQuery} from '../__generated__/AddTeamDialogQuery.graphql'
+import {AddTeamDialog_viewer$key} from '../__generated__/AddTeamDialog_viewer.graphql'
+import useMutationProps from '../hooks/useMutationProps'
+import useAtmosphere from '../hooks/useAtmosphere'
+import {AdhocTeamMultiSelect, Option} from '../components/AdhocTeamMultiSelect/AdhocTeamMultiSelect'
+import {Input} from '../ui/Input/Input'
+import AddTeamMutation from '~/mutations/AddTeamMutation'
+import useRouter from '~/hooks/useRouter'
+import getGraphQLError from '~/utils/relay/getGraphQLError'
+
+interface Props {
+  isOpen: boolean
+  onClose: () => void
+  onAddTeam: (teamId: string) => void
+  queryRef: PreloadedQuery<AddTeamDialogQuery>
+}
+
+const AddTeamDialogViewerFragment = graphql`
+  fragment AddTeamDialog_viewer on User {
+    ...AdhocTeamMultiSelect_viewer
+    organizations {
+      id
+      name
+    }
+  }
+`
+
+const query = graphql`
+  query AddTeamDialogQuery {
+    viewer {
+      ...AddTeamDialog_viewer
+    }
+  }
+`
+
+const AddTeamDialog = (props: Props) => {
+  const {isOpen, onClose, queryRef, onAddTeam} = props
+  const atmosphere = useAtmosphere()
+
+  const {submitting, onCompleted, onError, error, submitMutation} = useMutationProps()
+  const {history} = useRouter()
+
+  const data = usePreloadedQuery<AddTeamDialogQuery>(query, queryRef)
+  const viewer = useFragment<AddTeamDialog_viewer$key>(AddTeamDialogViewerFragment, data.viewer)
+  const {organizations: viewerOrganizations} = viewer
+
+  const labelStyles = `text-left text-sm font-semibold mb-3`
```

Comment:
+2 I like that you're using a var instead of repeating the code

+1 I don't think it needs to be in the component though. Could define it above `AddTeamDialog` to reduce the code inside it